app/models/shipit/webhooks.rb

```diff
@@ -5,6 +5,16 @@ class << self
       def default_handlers
         {
           'push' => [Handlers::PushHandler],
+          'pull_request' => [
+            Handlers::PullRequest::OpenedHandler,
+            Handlers::PullRequest::ClosedHandler,
+            Handlers::PullRequest::ReopenedHandler,
+            Handlers::PullRequest::EditedHandler,
+            Handlers::PullRequest::AssignedHandler,
+            Handlers::PullRequest::LabeledHandler,
+            Handlers::PullRequest::UnlabeledHandler,
+            Handlers::PullRequest::LabelCapturingHandler,
```

Comment:
Not a deal breaker, but I feel like there's lots of duplication in all these handlers, and that a single one handling all the events would be much more concise and easier to grasp.

But again, not a deal breaker, it can be refactored post merge in a follow up if there's no other blocker.